### PR TITLE
RD hardsuit built-in storage and general buff

### PR DESCRIPTION
**Pull request by Hmeister-real**
## About the PR
Changes the hardsuit to have a built in 3x3 storage, become a little faster, and also minorly increases it's stats.

## Why / Balance
With the recent (major) RD hardsuit nerf in #27094, I think it needs a buff, as the comments said further down in that PR. 

## Technical details
Stat changes and storage added

## Media

- [ ] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

:cl: Hmeister
- tweak: RD's hardsuit now has a built-in storage and is a little more protected

### CODE REVIEW

**Comment by Hmeister-real** [comment]
i have genuinely no clue how two checks failed, it is literally such a minor change. If someone has any idea how to fix it please let me know

File: Resources/Prototypes/Entities/Clothing/OuterClothing/hardsuits.yml [hunk 1]
```diff
@@ -382,10 +382,14 @@
...
-        Heat: 0.3
+        Piercing: 0.85
+        Heat: 0.25
         Radiation: 0.1
-        Caustic: 0.2
+        Caustic: 0.15
+  - type: Reflect
+        reflectProb: 0.55
+        reflects:
+           - Energy
```
Comment (EmoGarbage404) [inline Resources/Prototypes/Entities/Clothing/OuterClothing/hardsuits.yml:392]: indentation is fucked

Comment (Hmeister-real) [reply Resources/Prototypes/Entities/Clothing/OuterClothing/hardsuits.yml]: 👍

**Review by EmoGarbage404 — CHANGES_REQUESTED** [review]
I don't see why the RD suit, which is already extremely armored and resistant against explosions, needs to have higher amounts of combat-focused armor as well as reflective properties that almost solely impact laser combat.

It's already an extremely powerful bombsuit. I don't think there's really a reason for it to be much more.

**Comment by ps3moira** [comment]
power creeping the mysta hardsuit lol

**Comment by Hmeister-real** [comment]
> I don't see why the RD suit, which is already extremely armored and resistant against explosions, needs to have higher amounts of combat-focused armor as well as reflective properties that almost solely impact laser combat.

well the whole reason for it's experimental nature was the fact it was very portable and mobile, but with that removed it needs something unique to replace that. Any suggestions on another feature that would be better then?

**Comment by deltanedas** [comment]
maybe shitty ninja cloak that wears off after 5s with a big cooldown

**Comment by K-Dynamic** [comment]
Yeah what's up with this rd suit powercreep going on since the size increase

Just make it 3x3 or 2x4 storage, no need to make rd stronk

**Comment by Hmeister-real** [comment]
> maybe shitty ninja cloak that wears off after 5s with a big cooldown

Cloak is a syndie thing, don't think it would suit too well

> Just make it 3x3 or 2x4 storage, no need to make rd stronk

Im not touching the size since EmoGarbage merged #27094 with so little discussion. I assume that its a maintainer change and we have no choice in the matter :(

**Comment by BolloTea** [comment]
What if we gave the suit itself internal blast-proof storage of about 3x4 or 4x3?  This will make it more acceptable and not too op.  Yes, this is a small detail, but quite pleasant.  You can also make a built-in anomaly analyzer or a switchable node analyzer (sorry if I wrote the name wrong).  Still, this is primarily a scientific suit, not a combat one.

**Comment by K-Dynamic** [comment]
> What if we gave the suit itself internal blast-proof storage of about 3x4 or 4x3? This will make it more acceptable and not too op. Yes, this is a small detail, but quite pleasant. You can also make a built-in anomaly analyzer or a switchable node analyzer (sorry if I wrote the name wrong). Still, this is primarily a scientific suit, not a combat one.

I wouldn't be against this when we get hardsuit abilities some day

**Comment by Hmeister-real** [comment]
> What if we gave the suit itself internal blast-proof storage of about 3x4 or 4x3? This will make it more acceptable and not too op. Yes, this is a small detail, but quite pleasant. You can also make a built-in anomaly analyzer or a switchable node analyzer (sorry if I wrote the name wrong). Still, this is primarily a scientific suit, not a combat one.

I maybe could try at the storage idea but it would be very basic and just turn the suit into a bag too, but the built-in stuff is out of scope for this, it would be better for a PR adding suit abilities specifically

**Comment by Hmeister-real** [comment]
no clue why it failed, need help

**Comment by BolloTea** [comment]
By the suit's abilities, we mean switching modes, such as with the blood-red suit on the TG, where it has two modes?

**Comment by GoodWheatley** [comment]
Just remove the objective at this point

**Comment by HelpTheSteak** [comment]
Either remove the objective or unnerf the suit because the hardsuit is now the second hardest objective in the game

**Comment by DummeHytteOst** [comment]
> Either remove the objective or unnerf the suit because the hardsuit is now the second hardest objective in the game

the objective did have its difficulty increased since its become harder to get, so you likely wont have it with other high difficulty objectives. 

and as i commected here https://github.com/space-wizards/space-station-14/pull/27094#issuecomment-2066721381 i don't think the RD suit needs a buff, as it is incredibly good at what its supposed to be used for.

File: Resources/Prototypes/Entities/Clothing/OuterClothing/hardsuits.yml [hunk 2]
```diff
@@ -368,12 +368,15 @@
...
-  description: A special suit that protects against hazardous, low pressure environments. Has an additional layer of armor.
+  description: A special suit that protects against hazardous, low pressure environments. Has an additional layer of thick armor that doubles as a storage compartment.
   components:
   - type: Sprite
     sprite: Clothing/OuterClothing/Hardsuits/rd.rsi
   - type: Clothing
     sprite: Clothing/OuterClothing/Hardsuits/rd.rsi
+  - type: Storage
+    grid:
+  - 0,0,3,2
```
Comment (Tayrtahn) [inline Resources/Prototypes/Entities/Clothing/OuterClothing/hardsuits.yml:379]:
```suggestion
    grid:
    - 0,0,3,2
```

**Comment by Tayrtahn** [comment]
> > Either remove the objective or unnerf the suit because the hardsuit is now the second hardest objective in the game
> 
> the objective did have its difficulty increased since its become harder to get, so you likely wont have it with other high difficulty objectives. 
> 
> and as i commected here https://github.com/space-wizards/space-station-14/pull/27094#issuecomment-2066721381 i don't think the RD suit needs a buff, as it is incredibly good at what its supposed to be used for.

I agree with your linked comment. Having some drawbacks also makes it more likely that the RD will take it off and leave it in its locker like you said, which makes it more interesting as a steal objective. If the RD is always wearing it, it's effectively a kill objective.

**Comment by Partmedia** [comment]
Regrettably I'm opting to close this PR. As many commenters here have already pointed out, I feel that the RD hardsuit is already sufficiently strong that it does not need a "general buff" in response to a size increase in #27094. The size increase was made for the sole purpose of making this a sufficiently challenging steal objective and this "buff" feels like too much.

**Pull request closed without merging**